osbs/build/build_response.py

```diff
@@ -138,7 +138,20 @@ def get_error_reason(self):
 
         try:
             pod = self.osbs.get_pod_for_build(self.get_build_name())
-            return {'pod': pod.get_failure_reason()}
+            reason = pod.get_failure_reason()
+
+            if 'exitCode' in reason and 'containerID' in reason:
+                error_message = "oontainer {} has returned exit code {}, reason: {}".format(
+                                reason['containerID'], reason['exitCode'], reason['reason'])
+
+            elif 'exitCode' in reason:
+                error_message = "pod has returned exit code {}, reason: {}".format(
+                                reason['exitCode'], reason['reason'])
+
+            else:
+                error_message = "pod has failed, reason: {}".format(reason['reason'])
+
+            return {'pod': error_message}
```

Comment:
What is purpose of this change?

Reply:
@MartinBasti Hmm, you're right, this change doesn't concern current task, I'll remove this

Reply:
Original purpose of this commit was improve readability not using jsonschema as information origin

Reply:
We may (or may not I'm not sure yet) actually need something similar :D

Reply:
Not in in this PR though